python/lsst/ctrl/mpexec/cli/script/qgraph.py

```diff
@@ -42,6 +42,12 @@ def qgraph(pipelineObj, qgraph, skip_existing, save_qgraph, save_single_quanta,
     qgraph : `str` or `None`
         URI location for a serialized quantum graph definition as a pickle
         file. If this option is not None then `pipeline` should be `None`.
+    qgraph_id : `str` or `None`
+        Quantum graph identifier, graph loaded from a pickle file is required
+        to have this identifier. Ignored if graph is not loaded from a file.
```

Comment:
Did you want to change the wording here to match the updated help wording?

Reply:
OK, will do.